src/altinn-app-frontend/src/components/base/RadioButtonsContainerComponent.tsx

```diff
@@ -86,15 +86,16 @@ export const RadioButtonContainerComponent = ({
   formData,
   layout,
   legend,
-  title,
   shouldFocus,
   getTextResource,
   validationMessages,
   mapping,
   source,
+  likertDisplay,
+  componentValidations,
+  groupContainerId,
```

Comment:
The reason for this is to reference the headercolumn value of the current option of the radiobutton in LikertComponent.tsx line52 and 69:
```
// column label must reference correct id of header in table
const colLabelId = `${groupContainerId}-likert-columnheader-${colIndex}`;

<StyledRadio
                inputProps={{
                  'aria-labelledby': `${rowLabelId} ${colLabelId}`,
                  id: inputId,
                  role: 'radio',
                  name: rowLabelId,
                  'aria-checked': isChecked,
                }}
                checked={isChecked}
                onChange={handleChange}
                onBlur={handleBlur}
                value={option.value}
              />

```

Reply:
I could use the aria-label attribute instead of aria-labeledby, but this is not recommended to do when you have a visible html element you can reference.

Reply:
Yep, I see that we need the group container id, I would keep that as is :)